docs/deployments/api-configuration.md

```diff
@@ -58,6 +59,8 @@ See additional documentation for [autoscaling](autoscaling.md), [compute](comput
     signature_key: <string>  # name of the signature def to use for prediction (required if your model has more than one signature def)
     config: <string: value>  # arbitrary dictionary passed to the constructor of the Predictor (optional)
     python_path: <string>  # path to the root of your Python folder that will be appended to PYTHONPATH (default: folder containing cortex.yaml)
+    image: <string> # optional image to override the configured cluster api image (e.g. cortexlabs/tf-api)
```

Comment:
nit: "# docker image to use for the Predictor (default: cortexlabs/tf-api)"

Reply:
Done!